### ISSUE TITLE
Consider to increasing "desired retention" limits up from 0.97 and include 0.98 and 0.99.

### ISSUE THREAD

**Issue by Say-Hi-To-Me** [ankitects/anki#2803]
- I am ok with having a lot of reviews from desired retention as 0.98-0.99. And actually thinks that's been good because they would be way easier than usual because most of the cards are still in my memory and I didn't forget them yet. 

- So, in my opinion, desired retention at 0.98-0.99 would only save my time. Because I waste way more time on relearning than on reviewing those cards which I already knew.

- My personal experience only confirms my idea. 
- - At desired retention after switching to that new Anki FSRS's algorithm, I gradually increased desired retention by 0.01 and rescheduled and reviewed my cards. And then higher it was than easier it was review. 
- - Those cards which I got on review after setting desired retention at 0.97 and reschedule my deck was an easy walk. I actually enjoyed review cards at 0.97 there was no place for sadness and self-hatred because I don't forget and sent on relearning N-amount of cards as it was usually. 
- - - I actually saved my time at 0.97 by not wasting that time later on relearning as it usually was.

- High desired retention parameters like 0.98 and 0.99 only saved time and not wasting it.

- My proposal/idea is to increase that limit up to 0.99 included.

- For now, I find temporary solution in finding cards throughout deck browser with 98-97% retrievability (deck:current prop:r>=0.98 prop:r<=0.99) and manually sending them on reviews. I didn't practice that yet, though.
- - But I am sure that's not a serious solution in a long run for such tiny, really tiny problem which should not exist in the first place. Because I am happy at desired retention, 0.97, and I am sure 0.98 or 0.99 would not be a problem for me. And why someone need to set such limits on end user in the first place and remove any choice for him? Maybe someone need to prepare to an exam and be sure what he knew everything as perfect as possible and 1% is for him represent a huge difference.

---

TL;DR: please remove those weird limits at 0.97 desired retention up to 0.99 included.

Sorry for my oversimplistic English level.

Long Live Anki.
Long Live FOSS.

**Comment by dae** [comment]
I will defer to @L-M-Sherlock @expertium @user1823

**Comment by L-M-Sherlock** [comment]
According to our simulation, the workload increases drastically when the desired retention is higher than 0.97:

![](https://user-images.githubusercontent.com/83031600/280474267-5218e456-5ea9-408f-8f00-a1b5cb930de4.png)

Maybe we could loosen the limits but add a warning for that.

**Comment by Expertium** [comment]
@Say-Hi-To-Me the reason these boundaries exist is to prevent inefficient learning.
Below 0.7 time savings become negligible or even worse: you actually have to spend *more* time on Anki because you have to re-learning so many cards.
Above 0.97 the amount of time spent on Anki becomes extremely high. You are probably an outlier, I don't think most people would enjoy doing so many reviews.
Perhaps we should make a survey to ask lots of users about their desired retention. But right now not many people are using Anki 23.10, so I doubt that such a survey would get a lot of responses.

**Comment by user1823** [comment]
@Say-Hi-To-Me, as pointed out above, increasing the desired retention above 0.97 will greatly increase your review load.

There is also one more important disadvantage of setting a very high desirable retention — each review will do little to improve your learning. Let me quote a book titled "Make it Stick: The Science of Successful Learning":

> Psychologists have uncovered a curious inverse relationship between the ease of retrieval practice and the power of that practice to entrench learning: the easier a knowledge or a skill is for you to retrieve, the less your retrieval practice will benefit your retention of it. Conversely, the more effort you have to expend to retrieve knowledge or skill, the more the practice of retrieval will entrench it.
>
> [...]
>
> 1. Some difficulties that require more effort and slow down apparent gains—like spacing, interleaving, and mixing up the practice—will feel less productive at the time but will more than compensate for that by making the learning stronger, precise, and enduring.
> 2. Our judgments of what learning strategies work best for us are often distorted and coloured by illusions of mastery.

**Comment by Expertium** [comment]
Yes, in FSRS interval growth depends on R at the time of review (and other factors). The greater the value of R, the less the interval will increase if you successfully recall the card.

**Comment by Say-Hi-To-Me** [comment]
Thank you for your response, everyone! But why not let people decide that for themselves, as @L-M-Sherlock said: "Maybe we could loosen the limits but add a warning for that." @Expertium @user1823? With all respect, I just don't understand the reason behind limiting users, if someone wants to set the desired retention to 0.98/0.99, that is their personal business and not mine, right?

**Comment by Expertium** [comment]
This may sound arrogant, but sometimes users shoot themselves in the foot by choosing bad settings. Ideally, the people who are developing FSRS, such as Sherlock, should make it "fool-proof".

**Comment by user1823** [comment]
In my opinion, allowing setting desired retention up to 0.99 but adding a VERY STRONG warning will work.

**Comment by Expertium** [comment]
I disagree. At such high levels of desired retention, spaced repetition essentially turns into massed repetition, defeating the whole point of using Anki instead of just reviewing everything every day.

**Comment by Say-Hi-To-Me** [comment]
@Expertium you have a valid point.

But what if for example someone has an exam in front of him? 
And that person is ok with intentionally 'cramming' at 0.99? 
Many Anki user do such cramming intentionally with Custom Study.
Why not let them and me do the same cramming with 0.99? 

And it can be "fool-proof" by making a warning and/or hidden that for button like 'for advanced users'. 

I am not saying 0.99 is the best setting (or it must be a default option), my point is not about that. I'm just trying to say what there is a different life situations when a person is intentionally cramming before some event like exam or job interview material which is very important for that person.

Why not let them do that if they are ok with that and even without that they still are going to use cramming at Anki just differently either with custom study or with setting a lot of learning steps which is going to be the same 'crawl' with a lot of reviews?

**Comment by Expertium** [comment]
Ok, then let's allow it, but display the following warning (or a similar one, I'm bad at writing formal stuff):

"Setting the value of desired retention above 0.97 is not recommended. It will slow the growth of intervals while greatly increasing the number of daily reviews, effectively turning spaced repetition into massed repetition."

**Comment by user1823** [comment]
How about this one?

**Caution!** Desired retention above 0.97 may significantly increase your daily workload and diminish the benefits of spaced repetition. Please consider a lower value for an optimal learning experience.

**Comment by Expertium** [comment]
@dae see the above comment

**Comment by dae** [comment]
Instead of adding a warning, I've made Anki show the user how their DR would alter a 100 day interval, so they can see for themselves the effect it would have. The info box will change from blue to yellow to red as they approach 0.99. What do you think?

**Comment by user1823** [comment]
It is a good idea.

In addition to that, I suggest adding the following message (or something similar) when DR > 0.97:

With such high desired retention, your memories will paradoxically become very weak, leading to an even slower growth of intervals.

**Comment by Expertium** [comment]
> With such high desired retention, your memories will paradoxically become very weak, leading to an even slower growth of intervals.

I prefer the previous one, even though it's long.

**Comment by Say-Hi-To-Me** [comment]
@dae sorry for bothering you with my impatience, but when those changes are going to be applied? Is that going to be in version 23.11? Just I want to know approximately time which I need to wait like a month or what? Or I just need to redownload 23.10, and it's going to be different already?

**Comment by dae** [comment]
A new release will hopefully be out in a few days.

**Comment by Expertium** [comment]
@Say-Hi-To-Me Anki 23.10.1 has just been released, enjoy
